index.html

```diff
@@ -41,7 +41,16 @@ <h2 id="schedule">Schedule a demo today and deliver mondern mobile baking to you
         <a id="demo" href="https://bankjoy.com/#signup">Schedule a demo</a>
       </div>
 
-
+<!--Mobile app screenshots-->
+      <div id="phones">
+        <div id="back">
+          <img id="left" src="file:///Users/annie/Project-2-Bankjoy/left.png">
```

Comment:
These should just be the file names. You don't want the file:// schema or the local file path from your computer.